x/storage/keeper/payment.go

```diff
@@ -129,6 +145,25 @@ func (k Keeper) lockObjectStoreFee(ctx sdk.Context, bucketInfo *storagetypes.Buc
 		})
 	}
 
+	if ctx.IsUpgraded(upgradetypes.Pawnee) {
+		internalBucketInfo := k.MustGetInternalBucketInfo(ctx, bucketInfo.Id)
+		internalBucketInfo.PriceTime = timestamp
+		nextBill, err := k.GetBucketReadStoreBill(ctx, bucketInfo, internalBucketInfo)
```

Comment:
Most buckets do not have rate limit. Can we check if there is rate limiter earlier and break quickly? Otherwise, it seems it will affect the performance for the general cases.

Reply:
sure